playwright-csharp-xunit/PlaywrightTestParallel.cs

```diff
@@ -0,0 +1,63 @@
+namespace playwright_csharp_xunit
+{
+    using Microsoft.Playwright;
+    using Xunit;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class PlaywrightTestParallel
+    {
+        private static string GetCdpUrl()
+        {
+            string user, accessKey;
+            user = Environment.GetEnvironmentVariable("LT_USERNAME");
+            accessKey = Environment.GetEnvironmentVariable("LT_ACCESS_KEY");
+
+            Dictionary<string, object> capabilities = new Dictionary<string, object>();
+            Dictionary<string, string> ltOptions = new Dictionary<string, string>();
+
+            ltOptions.Add("name", "Playwright Test");
+            ltOptions.Add("build", "Playwright C-Sharp tests");
+            ltOptions.Add("platform", "Windows 10");
+            ltOptions.Add("user", user);
+            ltOptions.Add("accessKey", accessKey);
+
+            capabilities.Add("browserName", "Chrome");
```

Comment:
@alisengul-lambdatest add the FF and edge browser also.

Reply:
I already tried to add sample for Firefox and Safari but I got the error `Microsoft.Playwright.PlaywrightException: cdp-create-session: Requested browser:: Firefox not supported`
Are FF and edge supported for Playwright?

Reply:
edge is supperted in playwright

Reply:
Lets connect about it because I couldn't find configuration on csharp for Edge also. There a few things but none of them worked

Reply:
Added Microsoft edge also

Reply:
@ankur-lt kindly review 
LGTM